ui/js/controllers/perf/e10s-trend.js

```diff
@@ -1,487 +0,0 @@
-"use strict";
-
-perf.value('defaultBaseDate', 604800);
-perf.value('defaultNewDate', 0);
-perf.value('defaultSampleSize', 604800);
-
-perf.controller('e10sTrendCtrl', [
-    '$state', '$stateParams', '$scope', '$rootScope', '$q', '$httpParamSerializer',
-    'ThRepositoryModel', 'PhSeries', 'PhCompare',
-    'thDefaultRepo', 'phTimeRanges', 'defaultSampleSize',
-    'phComparisonDate', 'defaultBaseDate', 'defaultNewDate', 'phBlockers',
```

Comment:
`phComparisonDate` is now unused, so can be removed (if ever needed in the future, I'd prefer to import an ES6 style constant rather than use AngularJS `value()`s)

Reply:
Ok